quarkus/dist/src/main/content/bin/kc.sh

```diff
@@ -41,10 +41,9 @@ SERVER_OPTS="$SERVER_OPTS -Dpicocli.disable.closures=true"
 SERVER_OPTS="$SERVER_OPTS -Dquarkus-log-max-startup-records=10000"
 CLASSPATH_OPTS="'$(abs_path "../lib/quarkus-run.jar")'"
 
-DEBUG_MODE="${KC_DEBUG:-${DEBUG:-false}}"
```

Comment:
Just wanted to highlight this might be considered a breaking change – we no longer accept `KC_DEBUG` env var. It was not documented, though  – so it should be fine.

Reply:
That was not intentional.